src/client.sln

```diff
@@ -26,6 +26,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "endpointmanagerlauncher", "
 EndProject

```

Comment:
nit: changes on this file needed?

Reply:
Yes needed to run UT's via build systems, I am not sure why dsc.tests, routingmanager.tests, restoration.tests were excluded previously but I am going to add them and see how it goes.